sdkconfig.defaults

```diff
@@ -1,12 +1,16 @@
-CONFIG_PARTITION_TABLE_CUSTOM=y
-CONFIG_PARTITION_TABLE_CUSTOM_FILENAME="partitions.csv"
-CONFIG_PARTITION_TABLE_FILENAME="partitions.csv"
-CONFIG_PARTITION_TABLE_OFFSET=0x8000
-CONFIG_PARTITION_TABLE_MD5=y
+# This file was generated using idf.py save-defconfig. It can be edited manually.
+# Espressif IoT Development Framework (ESP-IDF) 5.3.1 Project Minimal Configuration
+#
+CONFIG_IDF_TARGET="esp32s3"
 CONFIG_ESPTOOLPY_FLASHSIZE_16MB=y
-CONFIG_ESPTOOLPY_FLASHSIZE="16MB"
+CONFIG_PARTITION_TABLE_CUSTOM=y
+CONFIG_COMPILER_ORPHAN_SECTIONS_WARNING=y
+CONFIG_HTTPD_MAX_URI_LEN=2048
 CONFIG_HTTPD_WS_SUPPORT=y
+CONFIG_SPIRAM=y
+CONFIG_SPIRAM_MODE_OCT=y
 CONFIG_SPIFFS_OBJ_NAME_LEN=64
-CONFIG_HTTPD_MAX_URI_LEN=2048
-CONFIG_ESP32S3_DEFAULT_CPU_FREQ_240=y
```

Comment:
You'll need to revert all the changes to sdkconfig.defaults and add your changes on top.

Reply:
There were two settings missing indeed:
CONFIG_ESP_DEFAULT_CPU_FREQ_MHZ_240
CONFIG_ESP_WIFI_11KV_SUPPORT

Some fields are removed as they have the default value:
CONFIG_PARTITION_TABLE_CUSTOM_FILENAME
CONFIG_PARTITION_TABLE_FILENAME
CONFIG_PARTITION_TABLE_OFFSET
CONFIG_PARTITION_TABLE_MD5

Some fields are superfluous, as they are already set by a different field:
CONFIG_ESPTOOLPY_FLASHSIZE (set by CONFIG_ESPTOOLPY_FLASHSIZE_16MB)

Some fields have a more general version:
CONFIG_ESP32S3_DEFAULT_CPU_FREQ_240 -> CONFIG_ESP_DEFAULT_CPU_FREQ_MHZ_240

So it's a bit of a clean-up.

The only thing I'm not sure about is the CONFIG_STRATUM_USER and CONFIG_FALLBACK_STRATUM_USER which are added to this file as well if you run save-defconfig. I don't think that should be happening?

I can do a complete revert and add new fields to the end, but I think this is cleaner.

Reply:
I would probably personally prefer to keep to only make the sdkconfig.default changes directly related to the display, but I don't feel strongly about it.

Reply:
I'll extract and make a separate issue for using save-defconfig and cleaning up `sdkconfig.defaults`.